docs/cloud/azureai/sharing-your-flows-in-promt-flow-gallery.md

```diff
@@ -0,0 +1,122 @@
+```
+  
+3. Require changes to spec.yaml:
+
+![spec yaml](../../media/cloud/azureml/flow_sharing/spec_yaml_in_azure_assets_repo.png)
+
+```yml
+name: provide a name for your promptflow model
+azureml.promptflow.type: value can be `chat`, `standard` or `evaluate`. This property identifies the type of your flow, and UI will display flows with  `evaluate` value under the 'Evaluation' tab, and display flows with other values under the 'Flow' tab.
+azureml.promptflow.name: the name of the flow which will be shown as the flow name in the Flow Gallery.
+azureml.promptflow.description: the description of the flow which will be shown as flow description in the Flow Gallery.
+version: the current version for your promptflow model.
+```
+
+4. Require change to description.md: need to change the model description, sample inputs and outputs part
+
+![description md](../../media/cloud/azureml/flow_sharing/description_in_azureml_assets_repo.png)
+
+5. Keep the asset.yaml as it is
+
+6. When the PR is ready, please send it to [azureml-assets channel](https://teams.microsoft.com/l/channel/19%3aeb1edc86fc204c07bc767b7bd7c8b091%40thread.tacv2/System%2520Registry%2520Assets?groupId=9afd1b1e-79ba-4975-9d50-912f53962917&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) to ask approve and merge
+
+#### Update an existing promptflow model:
+
+1. Change the container_path to include your new updated flow:
+
+![update container path](../../media/cloud/azureml/flow_sharing/update_container_path_in_azureml_assets.png)
+  
+2. Change the model version by increasing it by 1:
+
+![update model version](../../media/cloud/azureml/flow_sharing/update_model_version_in_azureml_assets.png)
+
+1. When the PR is ready, please send it to [azureml-assets channel](https://teams.microsoft.com/l/channel/19%3aeb1edc86fc204c07bc767b7bd7c8b091%40thread.tacv2/System%2520Registry%2520Assets?groupId=9afd1b1e-79ba-4975-9d50-912f53962917&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) to ask approve and merge
+
+### Preparing a PR to [azrueml asset ADO repo](https://msdata.visualstudio.com/Vienna/_git/azureml-asset?version=GBmain)
+
+> [!Note] This step is only required when there is new promptflow models added, please skip this step when you only update existing promptflow models.
+- Please refer [this PR](https://msdata.visualstudio.com/Vienna/_git/azureml-asset/pullrequest/1201381?_a=files) to add the new added promptflow model names to azureml-dev.yml, azureml-staging.yml and azureml.yml.
+- Send the PR to [azureml-assets channel](https://teams.microsoft.com/l/channel/19%3aeb1edc86fc204c07bc767b7bd7c8b091%40thread.tacv2/System%2520Registry%2520Assets?groupId=9afd1b1e-79ba-4975-9d50-912f53962917&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) to ask approve and merge
+
+## Registering your flow in AzureML Registry
+
+### Trigger [AzureML-Assets-Unified](https://dev.azure.com/msdata/Vienna/_build?definitionId=25490&_a=summary) to only build the new added models and the updated models
```

Comment:
Let's point to asset team reference https://msdata.visualstudio.com/Vienna/_wiki/wikis/vienna-wiki/54111/Build-Release-Process